lib/slack/lookups.ex

```diff
@@ -7,7 +7,7 @@ defmodule Slack.Lookups do
   def lookup_user_id("@" <> user_name, slack) do
     slack.users
     |> Map.values
-    |> Enum.find(%{}, fn user -> user.name == user_name end)
+    |> Enum.find(%{}, fn user -> user.name == user_name or user.profile.display_name == user_name end)
```

Comment:
nice, I think `profile.display_name` is a relatively new thing. This is a great fix. One minor nitpick, can we change `or` to `||`?

Reply:
Yep, I can change it.

There is similar code at:

https://github.com/BlakeWilliams/Elixir-Slack/blob/58ffa73559b5cb29d41c2a472ee3c4ba53ed6333/lib/slack/lookups.ex#L27

You might want to check if we should change it as well.

Reply:
Yep, I believe we'll have to change that there as well.